jetty-ee9/jetty-ee9-security/src/main/java/org/eclipse/jetty/ee9/security/PropertyUserStore.java

```diff
@@ -159,13 +143,14 @@ protected void loadUsers() throws IOException
         if (LOG.isDebugEnabled())
             LOG.debug("Loading {} from {}", this, config);
 
-        if (!config.exists())
+        if (Resources.missing(config))
             throw new IllegalStateException("Config does not exist: " + config);
 
         Properties properties = new Properties();
         try (InputStream inputStream = config.newInputStream())
         {
-            properties.load(inputStream);
+            if (inputStream != null)
+                properties.load(inputStream);
```

Comment:
I think you should add an else branch that throws `new IllegalStateException("Config cannot be read: " + config)` or something similar.